ci/sge/run-master.sh

```diff
@@ -20,4 +20,8 @@ bash add_worker.sh dask.q slave_two 4
 
 sudo service gridengine-master restart
 
-python -m http.server 8888
+if [ "$TRAVIS_PYTHON_VERSION" == "2.7" ]; then
+    python -m SimpleHTTPServer 8888
+else
+    python -m http.server 8888
+fi
```

Comment:
Launching this http server is just in order to have a blocking process when running the docker container. I don't really like complexifying this line, even if I don't have a better solution yet (maybe just use some blocking commands, see https://stackoverflow.com/questions/2935183/bash-infinite-sleep-infinite-blocking).

Reply:
I'll try out some variants based on above and can hopefully remove both above conditional and Docker ENV